tools/wpt/tests/test_run.py

```diff
@@ -16,3 +22,22 @@ def test_check_environ_fail(platform):
                 run.check_environ("foo")
 
     assert "wpt make-hosts-file" in excinfo.value.message
+
+
+@pytest.mark.parametrize("product", product_list)
+def test_setup_wptrunner(product):
```

Comment:
This test seems to change some global state, making later tests fail.

`tox -e py27 -- --no-cov -k 'test_run_firefox or test_setup_wptrunner'` is sufficient to observe a failure; Travis has many more.